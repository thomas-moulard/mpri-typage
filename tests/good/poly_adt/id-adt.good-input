type id 'a =
  Id of 'a
end

val f x =
  match x with
    | Id y -> y
  end
