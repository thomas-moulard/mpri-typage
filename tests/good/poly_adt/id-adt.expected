Id = Id

f = fun x.match x with 
    | Id y -> y
end


f : forall 'a.(id 'a -> 'a)
Id : forall 'a.('a -> id 'a)
